### PR TITLE
Provision ocp cluster

### PR DESCRIPTION
**Pull request by rayashworth**
adding cluster provisioning play ( AWS  only) via ACM/Hive instance.

### CODE REVIEW

File: ansible/provision-ocp-cluster-play/examples/google-vars.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"google_project": "YOUR_GOOGLE_PROJECT"   # example: oceanic-guard-191815
+"google_zone": "YOUR_GOOGLE_ZONE"
+"google_auth_kind: "serviceaccount"
```
Comment (websterc87) [inline ansible/provision-ocp-cluster-play/examples/google-vars.yml:4]: Is this line missing a quote?

**Review by websterc87 — CHANGES_REQUESTED** [review]
Lots of content in the PR, makes it hard to review with any degree of certaintcy, I have tried to spot any obvious errors. Its generally looks good besides the one missing quote.

Assuming you have tested running against AWS this seems good to deliver. Bugs can be fixed when/if they are found.

File: ansible/provision-ocp-cluster-play/examples/google-vars.yml (hunk 1 above)
Comment (rayashworth) [reply ansible/provision-ocp-cluster-play/examples/google-vars.yml]: quote added and pushed

**Review by wkrapohl — APPROVED** [review]
Go to go

**Pull request merged**